website/docs/components/card/partials/code/how-to-use.md

```diff
@@ -18,6 +18,28 @@ To style the Cards, you can add an external element that wraps the Card, with a
 
 Alternatively, you could use the Card Containers in a CSS `flex` or `grid` container.
 
+### HTML tag
+
+To specify which HTML tag to use to render the component, use the `@tag` argument. The default tag is a `div` but you can optionally render the Card as an `li` to be used within a list.
+
+Note: If you choose to use the `Card` as a list item, you must wrap it either in a `ul` or `ol` tag for the markup to be valid. Also note that you are responsible for the related styling for the list and list items.
```

Comment:
Could also put this in an info banner, but I'm not opinionated.

Reply:
Let me look at tweaking it after release if needed. Thanks